MAUI/Scheduler/appointments.md

```diff
@@ -187,6 +187,8 @@ this.Scheduler.AppointmentsSource = Meetings;
 ![business-appointment-event-in-maui-scheduler](images/appointments/business-appointment-event-in-maui-scheduler.png)
 
 N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main/BusinessObject)
+N> When publishing in AOT mode on iOS, ensure `[Preserve(AllMembers = true)]` is added to the model class to maintain appointment binding
```

Comment:
When publishing in AOT mode on iOS, ensure that [Preserve(AllMembers = true)] is added to the model class to maintain appointment binding.